tests/desktop/consumer_pages/test_details_page.py

```diff
@@ -14,6 +14,9 @@ class TestDetailsPage(BaseTest):
     @pytest.mark.sanity
     @pytest.mark.nondestructive
     def test_that_application_page_contains_proper_objects(self, mozwebqa):
+        if '-dev' in mozwebqa.base_url:
+            pytest.xfail("Bug 1242648 - Support info doesn't show up on an empty profile")
```

Comment:
Nit: This is an imperative xfail that could never xpass. If you implement it as a marker with a condition then it could xpass. See http://pytest.org/latest/skipping.html#mark-a-test-function-as-expected-to-fail for more information.